tedana/workflows/tedana.py

```diff
@@ -618,7 +622,7 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     comp_names = comptable["Component"].values
     mixing_df = pd.DataFrame(data=mmix, columns=comp_names)
     mixing_df.to_csv(io_generator.get_name("ICA mixing tsv"), sep="\t", index=False)
-    betas_oc = utils.unmask(computefeats2(data_oc, mmix, mask), mask)
+    betas_oc = utils.unmask(computefeats2(data_oc, mmix, mask_clf), mask_clf)
```

Comment:
These are used for visualization, not classification, so it might be better to use `mask_denoise` here, but I'm not sure.

Reply:
I would agree with using `mask_denoise` here since it's for visualization, but maybe @handwerkerd has some thoughts?

Reply:
I agree.

Reply:
Thanks! Handled in 8317ff1.